SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -82,6 +82,16 @@ class SlidingSegmentedControlTests: XCTestCase {
         XCTAssertEqual(slidingSegmentedControl.imageViews.map {$0.contentMode}, [.center, .center, .center])
     }
 
+    func testImageViewsIsAccessibilityElement() {
+        let slidingSegmentedControl = SlidingSegmentedControl(images: [UIImage(), UIImage()])
+        XCTAssertEqual(slidingSegmentedControl.imageViews.map { $0.isAccessibilityElement }, [true, true])
```

Comment:
@salutis 
That works! But 4 lines of tests replaced by 18. Man, you really must be hating those `flatMap`s 😄 

Getting rid of them as promised.